[DOCS-8936] adding DNS reverse IP enrichment to Netflow (#27965)
DataDog/documentation: content/en/network_monitoring/netflow/_index.md

{{< img src="network_device_monitoring/netflow/new_enrichment.png" alt="The New Enrichment Mapping modal in the Netflow configuration tab" width="80%" >}}

### Reverse DNS private IP enrichment

> Enable Reverse DNS private IP enrichment to perform DNS lookups and find the domain name associated with any NetFlow source or destination IP address. When enabled, the Agent conducts reverse DNS lookups on source and/or destination IPs within private address ranges, enriching NetFlow records with the corresponding hostnames.

Reviewer: "domain name" is mentioned initially, then "hostnames" at the end.  We should probably stick with one, IMO hostname.

"any NetFlow source or destination IP address" - saying "any" may be misleading since we only attempt reverse DNS lookups for private source/dest IP addresses (that are in the reserved private IP address ranges https://www.arin.net/reference/research/statistics/address_filters/)

> Add the following to your `datadog.yaml` file to enable Reverse DNS IP enrichment (disabled by [default][7]):

Reviewer: Would it make sense to have this in the main Configuration section, above?  Primarily so the proper indentation would be clear.

{{< img src="network_device_monitoring/netflow/new_enrichment.png" alt="The New Enrichment Mapping modal in the Netflow configuration tab" width="80%" >}}

### Reverse DNS private IP enrichment

> Enable Reverse DNS private IP enrichment to perform DNS lookups to find the hostname name associated with source or destination IP addresses. When enabled, the Agent conducts reverse DNS lookups on source and/or destination IPs within private address ranges, enriching NetFlow records with the corresponding hostnames.

Reviewer 2: ```suggestion
Enable Reverse DNS private IP enrichment to perform DNS lookups for hostnames associated with source or destination IP addresses. When enabled, the Agent conducts reverse DNS lookups on source and destination IPs within private address ranges, enriching NetFlow records with the corresponding hostnames.
```

> Reference the [configuration](#configuration) file above to enable Reverse DNS IP enrichment in your `datadog.yaml` file (disabled by [default][7]):

Reviewer 2: ```suggestion
By [default][7], the Reverse DNS IP enrichment in your `datadog.yaml` file is disabled. To enable, see the [Configuration](#configuration) section of this page.
```

> Search for **DNS** in the Flow grouping of the facets section to locate flows associated with reverse DNS IP enrichment: 

Reviewer 2: Is it reverse DNS IP enrichment or Reverse DNS IP enrichment? It's capitalized in other places, but not here.

{{< img src="network_device_monitoring/netflow/dns_ip_enrichment.png" alt="Screenshot of the reverse DNS destination and source facets" width="100%" >}}

> **Note**: Reverse DNS entries are cached and subject to rate limiting to minimize DNS queries and reduce the load on DNS servers. For more configuration options, including modifying default caching and rate limiting, reference the [full configuration file][8].

Reviewer 2: ```suggestion
**Note**: Reverse DNS entries are cached and subject to rate limiting to minimize DNS queries and reduce the load on DNS servers. For more configuration options, including modifying default caching and rate limiting, see the [full configuration file][8].
```

| Destination Subdivision ISO Code | The ISO code representing the subdivision (such as state or province) associated with the destination IP. |
| Destination Subdivision Name | The name of the subdivision (such as state or province) associated with the destination IP. |
| Destination Timezone | The timezone associated with the destination IP. |

> | Destination Reverse DNS Hostname | The DNS hostname associated with the destination IP. |

Reviewer: Should this be moved up so it's in alphabetical order?
